tools/azure-pipelines/build-nightly-dist.yml

```diff
@@ -109,7 +109,7 @@ jobs:
             </settings>
             EOF
 
-            export CUSTOM_OPTIONS="-Dgpg.skip -Drat.skip -Dcheckstyle.skip -Dmaven.wagon.http.pool=false --settings $(pwd)/deploy-settings.xml"
+            export CUSTOM_OPTIONS="${MVN_GLOBAL_OPTIONS_WITHOUT_MIRROR} -Dgpg.skip -Drat.skip -Dcheckstyle.skip --settings $(pwd)/deploy-settings.xml"
```

Comment:
Is it intended that we propagate some options that were not there before? Logging was mentioned in the description, but `--no-snapshot updates -B $PROFILE` is now also applied in https://github.com/apache/flink/blob/master/tools/releasing/deploy_staging_jars.sh#L46-L50

Reply:
hmm..... -B makes no difference, -nsu is good because it can highlight mistakes in the dependency setup. $PROFILE could be problematic because it contains the scala switch :/

Reply:
$PROFILE should be fine as it does not contain anything at the moment, and could be useful at some point for parameterization.